JaraR/DarkPatternDetector
# Create branch design and Add initial file

## Description
Created the `design` branch to track of the version of the prototype UI design. This should allow every group member to access it and provide any feedback they may have.


## Changes Made
- [x] Created the `design` branch.
- [x] Added `delete-me.md` as a placeholder in the `figma-prototype-docs` folder.

## Testing Instructions
Open the Figma prototype files and ensure that they align with the project requirements.
